lib/packwerk/graph.rb

```diff
@@ -4,8 +4,14 @@
 module Packwerk
   # A general implementation of a graph data structure with the ability to check for - and list - cycles.
   class Graph
-    # @param [Array<Array>] edges The edges of the graph; An edge being represented as an Array of two nodes.
-    def initialize(*edges)
```

Comment:
Splats are poorly supported by sorbet and I didn't see a great reason to keep this as a splat, but let me know if I'm missing something.